multimodal/src/autogluon/multimodal/utils/model.py

```diff
@@ -523,3 +524,47 @@ def modify_duplicate_model_names(
 
 def list_timm_models(pretrained=True):
     return timm.list_models(pretrained=pretrained)
+
+
+def _filter_timm_pretrained_cfg(cfg, remove_source=False, remove_null=True):
```

Comment:
Since `_filter_timm_pretrained_cfg ` and `save_timm_config ` are about config, is it better to put them into `utils/config.py`?